src/js/main.js

```diff
@@ -0,0 +1,59 @@
+const form = document.getElementById('registrationForm');
+const fullName = document.getElementById('fullName');
```

Comment:
your global scope is polluted with lots of variables. Consider using IFFY: https://codeburst.io/javascript-what-the-heck-is-an-immediately-invoked-function-expression-a0ed32b66c18